how-to-guides/12-lr-scheduler.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "<!-- ---\n",
+        "title: <required-title>\n",
+        "date: 2022-03-11\n",
+        "downloads: true\n",
+        "weight: 12\n",
+        "sidebar: True\n",
+        "summary: This example demonstrates how to use the [torch.optim.lr_scheduler](https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.LambdaLR) to adjust the learning rate of a model.\n",
+        "tags:\n",
+        "  - lr scheduler\n",
+        "--- -->\n",
+        "\n",
+        "# How to use LR-Schedulers"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This how-to guide demonstrates how we can use LR-Schedulers to adjust the learning rate of a model."
```

Comment:
@divyanshugit please add more context and links on ignite docs for the functionalities used in the NB.